components/match2/wikis/osu/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,108 @@
+---
+-- @Liquipedia
+-- wiki=osu
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local BaseCopyPaste = Lua.import('Module:GetMatchGroupCopyPaste/wiki/Base')
+
+---@class OsuMatch2CopyPaste: Match2CopyPasteBase
+local WikiCopyPaste = Class.new(BaseCopyPaste)
+
+local VETOES = {
+	[0] = '',
+	[1] = 'ban,ban,ban,decider',
+	[2] = 'ban,ban,pick,ban',
+	[3] = 'ban,pick,ban,decider',
+	[5] = 'ban,pick,pick,decider',
+	[7] = 'ban,pick,pick,pick,decider',
+	[9] = 'ban,pick,pick,pick,pick,decider',
+	[11] = 'ban,pick,pick,pick,pick,pick,decider',
+	[13] = 'ban,pick,pick,pick,pick,pick,pick,decider',
```

Comment:
```suggestion
	[1] = {'ban', 'ban', 'ban', 'decider'},
	[2] = {'ban', 'ban', 'pick', 'ban'},
	[3] = {'ban', 'pick', 'ban', 'decider'},
	[5] = {'ban', 'pick', 'pick', 'decider'},
	[7] = {'ban', 'pick', 'pick', 'pick', 'decider'},
	[9] = {'ban', 'pick', 'pick', 'pick', 'pick', 'decider'},
	[11] = {'ban', 'pick', 'pick', 'pick', 'pick', 'pick', 'decider'},
	[13] = {'ban', 'pick', 'pick', 'pick', 'pick', 'pick', 'pick', 'decider'},
```
please double check this!

(if bestof 0 together with protect should be available then also add `[0] = {},`)